Dockerfile

```diff
@@ -1,7 +1,7 @@
 # 
 # BUILDER_BASE is a multi-platform image with all the build tools
 #
-ARG BUILDER_BASE=quay.io/cilium/cilium-envoy-builder:6.0.0-latest
+ARG BUILDER_BASE=quay.io/cilium/cilium-envoy-builder:6.0.0-latest@sha256:cc23a75e28e3deda8cf1f486ef788c4fdf2f21c9e88dc3cdcd81d271abf43c84
```

Comment:
@sayboras seems to be related with the changes of https://github.com/cilium/proxy/pull/187

i think it's ok to pin it. otherwise i can add an renovate exception. WDYT?

Reply:
yeah, I think it's ok to pin it :+1:

Reply:
do you think if there is any cyclic dependency here ?

Reply:
@sayboras 

it should only change if the bazelversion (`6.0.0`) or content of `Dockerfile.builder` changes. therefore noise factor shouildn't the problem.

but you're right that there might be problems when upgrading the bazelversion - currently after merging it into the main branch, the "release" build would use the latest image from its own run. This would no longer be the case.

but i kind of wonder how this usecase of upgrading the bazelversion works in the PR branch where `latest` with a new bazelversion is expected - which itself only gets pushed as "latest" in the main branch :thinking: i would expect a build error